src/javascript/__init__.py

```diff
@@ -20,6 +27,11 @@ def init():
 init()
 
 
+def terminate():
```

Comment:
Can you add a note for this and the other new API method to the docs along with an explanation or how to use?